.evergreen.yml

```diff
@@ -204,7 +204,7 @@ functions:
           # https://docs.google.com/document/d/1IfQGC7wTtrlsc2SqURirvt_4uMuU606nXNbu-stw6bQ/edit
           # 
           # test_suite option only affects compass tests
-          DEBUG=* MONGODB_VERSION=${mongodb_version|4.0.3} npm run test-evergreen -- -- --${test_suite}
+          DEBUG=mongo*,mocha*,hadron* MONGODB_VERSION=${mongodb_version|4.0.3} npm run test-evergreen
```

Comment:
So `DEBUG=` seemed to be the reason for a lot of test failure, probably due to the sheer amount of stuff being written to stdout messing up the execution somehow

I also removed the `-- -- --${test_suite}` part because it was not doing anything

Reply:
Sounds good - if later we find those extra logs could be helpful for diagnosing failing test we can find a way to re-enable them without breaking execution.